libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/MapboxSpeechPlayer.java

```diff
@@ -152,18 +152,23 @@ public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response)
         if (response.isSuccessful()) {
           executeInstructionTask(response.body());
         } else {
-          onError();
+          try {
+            onError(response.errorBody().string());
+          } catch (IOException exception) {
+            onError("");
```

Comment:
What about `exception.getMessage()` instead?

Reply:
@Guardiola31337 That's the IOException from trying to read the error body, do you think that's relevant?

Reply:
Yeap, actually good to know that the `IOException` was the error cause (always better than nothing `""` 😅).

Reply:
@Guardiola31337 but that's not the actual error that caused `onError` to be called, it's an error retrieving the error 😓 . Still log it anyway?

Reply:
@devotaaabel I'd say yes. As I said if we're going to add error logs I think is still interesting to differentiate those two.